.github/scripts/lint-pr-title.cjs

```diff
@@ -0,0 +1,106 @@
+/**
+ * This Script can be tested locally with
+ *
+ * export GH_TOKEN=**** GITHUB_REPOSITORY=quisquous/cactbot PR_NUMBER=$NUM
+ * node ./.github/scripts/lint-pr-title.cjs
+ */
+'use strict';
+
+const github = require('@actions/github');
+
+const validScope = [
+  'resources',
+  'ui',
+  'pullcounter',
+  'raidboss',
+  'raidemulater',
+  'oopsy',
+  'radar',
+  'eureka',
+  'jobs',
+  'test',
+  'util',
+  'config',
+  'lint',
+  'build',
+  'ci',
+];
+
+const getComment = (title, userName) => `${userName ? '@' + userName : ''} Thanks for your contribution!🚀
```

Comment:
Does this have an extra leading space when there's not a userName? When is there not a userName?

Reply:
Yes.
I don't know, but the type hint from github' package said `user` can be an `object` or `null`.